rclpy/executors/examples_rclpy_executors/custom_executor.py

```diff
@@ -45,16 +45,14 @@ class PriorityExecutor(Executor):
     def __init__(self):
         super().__init__()
         self.high_priority_nodes = set()
-        self.low_priority_thread = None
+        self.hp_executor = ThreadPoolExecutor(max_workers=10)
```

Comment:
I know it is only an example... but `10` seems a little high. Maybe `cpu_count()`?

Reply:
018513a66e1f025c26ea76112b91f0122090498b